src/utils/form.ts

```diff
@@ -0,0 +1,10 @@
+export function submitClosestForm(element: HTMLElement) {
```

Comment:
call method like form.submit() ?
I have tried using that but it will not call the handleSubmit method on the form. It seems that programmatically calling form submit will result in different behavior than calling it using button.

which is why I use that way.

I can send additional props to the textArea like onFormSubmit, but I think this way is better because it will encapsulate the behavior on the textarea itself.

Reply:
I afraid that it really unclear, and strange for React project, rewrite please it to props. I understood your point, and agree, but I think about the project as project which will reused of other devs